igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java

```diff
@@ -48,6 +52,24 @@
     return parse(inputString, destinationClass);
   }
 
+  @SuppressWarnings("unchecked")
+  public final Map serializeBuild(Build inputBuild) {
+    // com.google.api.services.cloudbuild.v1.model.StorageSource.generation is of type Long
```

Comment:
@plumpy think you could find the right team to let them know about this?

Reply:
Still an issue but moved the fix in test code as this only impacts our tests so far.
https://github.com/spinnaker/igor/blob/c467ba2ad813c3b4abf4e0ab965ab020edbdeb86/igor-web/src/test/groovy/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildTestSerializationHelper.groovy